src/SolverAPI.jl

```diff
@@ -95,21 +95,22 @@ function response(json::Request, model::MOI.ModelLike; version = "0.1", kw...)
     end
 
     result_count = MOI.get(model, MOI.ResultCount())
-
     results = [Dict{String,Any}() for _ in 1:result_count]
+    var_names = [string('\"', v, '\"') for v in json.variables]
+    var_idxs = MOI.get(model, MOI.ListOfVariableIndices())
 
     for idx in 1:result_count
-        results[idx]["primal_status"] = MOI.get(model, MOI.PrimalStatus(idx))
+        res_idx = results[idx]
```

Comment:
Can we rename `res_idx` to `result`? Or some other name? `res_idx` for me indicates that the variable references an index, even though it references a dictionary.